zulipterminal/model.py

```diff
@@ -180,6 +180,7 @@ def __init__(self, controller: Any) -> None:
         self.active_emoji_data = self.generate_all_emoji_data(
             self.initial_data["realm_emoji"]
         )
+        self.all_emoji_names = self.generate_all_emoji_names()
```

Comment:
We update emoji when we get a realm emoji change event, so we at least want to call both in each case, or have one method that returns a tuple and updates both at the same time.